.Gruntfile.babel.js

```diff
@@ -125,7 +125,7 @@ module.exports = (grunt) => {
       sbp('backend/pubsub/setup', require('http').createServer(), true)
     }
     if (!grunt.option('skipbuild')) {
-      grunt.task.run(['exec:eslint', 'exec:puglint', 'exec:stylelint', 'copy', rollup])
+      grunt.task.run(['exec:eslint', 'exec:flow', 'exec:puglint', 'exec:stylelint', 'copy', rollup])
```

Comment:
The issue with adding it to the build step is that it will slow down development on first run. We can maybe add it here once we close #1043, but for now a better place might be as part of running `grunt test`.

Also, I'm not sure we're ready yet to have it as a requirement. One way to ignore any failures from the command is to add `|| true` after it (although flow might have a command line flag to not fail on error too, which would be better to use if such a flag exists).

Reply:
Agreed, let's not do type-checking when no file has changed yet.
I had mainly included Flow here for consistency, because of the three linting tasks that were already listed.

Reply:
Besides, I'm too in favor of a progressive integration rather than a radical change in the requirements.
But I'm hesitating regarding the best place to include `exec:flow` inside the `grunt test` and/or `grunt test:unit` task suites.
Should it run after or before the actual tests? I can only tell that if a failed Flow check gets ignored, then it would better run last, so that its console output has a chance to actually be read by the user.

Reply:
Hmm, that's a great question, and I'm not sure as to the answer.

The problem with running it after the tests is that we want to make sure that it *isn't* run if any of the tests fail, so that the program exits with a non-zero exit code and tells Travis things are broken.

And on the other hand, in most traditional development flows, these kinds of checks would run first thing, before the program ran. So maybe where you've placed it here is perfect. We can remove it in the future if it starts causing problems. And since you've actually fixed all the flow errors, maybe this is good hygiene for us to avoid massive PRs in the future as the flow errors diverge from the codebase.

So I'm going to resolve this thread. Let's keep it here for now.